frontend_app_kanban/i18next.scanner/de/translation.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Error while saving the comment": "__NOT_TRANSLATED__",
```

Comment:
Reminder to handle the translations before merging

Reply:
"Error while saving the comment": "Fehler beim Speichern des Kommentars",  
  "Delete": "Löschen",
  "The content has been modified by {{author}}": "Der Inhalt wurde von {{author}} geändertdrive",
  "New Kanban": "Neuer Kanban",
  "A content with the same name already exists": "Ein Inhalt mit demselben Namen existiert bereit",
  "Error while creating kanban": "Fehler beim Erstellen eines Kanbans",
  "Validate and create": "Validieren und erstellen",
  "Topic's subject": "__NOT_TRANSLATED__"

Reply:
What does "Topic's subject" mean in this context? I'm confused by this wording